configurations/generic/credhub.yml

```diff
@@ -2,9 +2,9 @@
   path: /releases/-
```

Comment:
The only reason we didn't switch over to `bosh-deployment/credhub.yml` was because of the version. Once they update on their end we should be able to switch over no problem. The only real difference is variable names (e.g. credhub-user vs. credhub-cli). If we switch those over we'll have to go through and make those changes everywhere so keeping our existing variable names seemed less disruptive.